src/renderer/views/Popular/Popular.vue

```diff
@@ -13,6 +13,13 @@
         :data="shownResults"
       />
     </ft-card>
+    <ft-icon-button
+      icon="sync"
+      class="floatingTopButton"
+      :size="12"
+      :theme="primary"
```

Comment:
This colon will need to be removed as adding it tells Vue to search for a `primary` variable.  Removing the colon will send over a string with the value `"primary"`, which is what we want.